src/Compiler/FSComp.txt

```diff
@@ -1595,6 +1595,7 @@ featureUnionIsPropertiesVisible,"Union case test properties"
 featureBooleanReturningAndReturnTypeDirectedPartialActivePattern,"Boolean-returning and return-type-directed partial active patterns"
 featureEnforceAttributeTargetsOnFunctions,"Enforce AttributeTargets on functions"
 featureEnforceAttributeTargetsUnionCaseDeclarations,"Enforce AttributeTargets on union case declarations"
+featureEnforceAttributeTargetsOnImplicitConstructors,"Enforce AttributeTargets on implicit constructors"
```

Comment:
Now I think we could have probably used one flag for all of these - but, doesn't matter much :)